tests/particle_distribution_statistics.prm

```diff
@@ -0,0 +1,155 @@
+# Adapted from the drop-supg.prm benchmark
```

Comment:
Can you add one more test that uses the KDE per particle feature and maybe a different kernel function (my favourite is the deal.II CutoffW1)?